code/modules/events/spider_infestation.dm

```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/spider_infestation
 	name = "Spider Infestation"
 	typepath = /datum/round_event/spider_infestation
-	weight = 5
+	average_time = 45

```

Comment:
Spiders aren't quite as dangerous as aliens, given that they're never player-controlled without admin intervention.